crates/dojo-types/src/primitive.rs

```diff
@@ -126,10 +145,15 @@ impl Primitive {
             Primitive::U128(_) => 4,
             Primitive::U256(_) => 5,
             Primitive::USize(_) => 6,
-            Primitive::Bool(_) => 7,
-            Primitive::Felt252(_) => 8,
-            Primitive::ClassHash(_) => 9,
-            Primitive::ContractAddress(_) => 10,
+            Primitive::I8(_) => 7,
+            Primitive::I16(_) => 8,
+            Primitive::I32(_) => 9,
+            Primitive::I64(_) => 10,
+            Primitive::I128(_) => 11,
+            Primitive::Bool(_) => 12,
+            Primitive::Felt252(_) => 13,
+            Primitive::ClassHash(_) => 14,
+            Primitive::ContractAddress(_) => 15,
```

Comment:
Here we shouldn't change the order as it's already used in other places. Could you simply append the new variants increasing the number please?

Reply:
Okay